client/js/game.js

```diff
@@ -23463,7 +23477,7 @@ function Game() {
             mode: undefined,
         };
         var tid=Math.floor(Date.now()/(24*60*60*1000));
-        var events=["lucky followers","cc","pge","lottery","flash","swb","dungeon","key tower","adventure"];
+        var events=["lucky followers","cc","pge","lottery","flash","eas","dungeon","key tower","adventure"];
```

Comment:
Is that ever used?

Reply:
(PR seems fine otherwise)

Reply:
I don't think so but wanted to clarify ^^